autoscheduler/frontend/src/.eslintignore

```diff
@@ -0,0 +1 @@
+**/*.css.d.ts
```

Comment:
The new CSS loader is not very ESLint-friendly, so I just disabled linting entirely